Modules/Core/Filament/Admin/Resources/ReportBlocks/Tables/ReportBlocksTable.php

```
@@ -0,0 +1,49 @@
+                IconColumn::make('is_active')
+                    ->boolean()
+                    ->sortable(),
+                IconColumn::make('is_system')
+                    ->boolean()
+                    ->sortable(),
+            ])
+            ->recordActions([
+                ActionGroup::make([
+                    EditAction::make(),
+                    DeleteAction::make('delete')
+                        ->action(function (ReportBlock $record, array $data) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$data'. (undefined)

(UnusedFormalParameter)